frigate/config.py

```diff
@@ -920,9 +921,10 @@ def verify_recording_segments_setup_with_reasonable_time(
             f"Camera {camera_config.name} has no segment_time in recording output args, segment args are required for record."
         )
 
-    if int(record_args[seg_arg_index + 1]) > 60:
+    config_duration = int(record_args[seg_arg_index + 1])
+    if config_duration < MIN_SEGMENT_DURATION or config_duration > 60:
         raise ValueError(
-            f"Camera {camera_config.name} has invalid segment_time output arg, segment_time must be 60 or less."
+            f"Camera {camera_config.name} has invalid segment_time output arg, segment_time must be {MIN_SEGMENT_DURATION} <= x <= 60."
```

Comment:
I would use MAX_SEGMENT_DURATION instead of 60 here since you already have it, and then I would also unify the units of min/max segment duration, they  look to be mismatched or something with MAX being set to 600?

Reply:
No, these are different on purpose. Users are only allowed to specify 60 seconds of segment timeout. However, ffmpeg is not always able to accomplish that based on the cameras iframe interval so a segment with up to 600 seconds in duration is acceptable to ensure valid segments are not accidentally dropped

Reply:
I see, replacing the 60 with an appropriately named constant might still be a good idea, you would know best though!

Reply:
I see how that is confusing and I thought about it while working on it but decided against it, makes sense to use a const